NewApp/ngsa-csharp/Ngsa.DataService/Controllers/MoviesController.cs

```diff
@@ -43,13 +49,26 @@ public async Task<IActionResult> GetMoviesAsync([FromQuery] MovieQueryParameters
                 throw new ArgumentNullException(nameof(movieQueryParameters));
             }
 
+            NgsaLog nLogger = Logger.GetLogger(nameof(GetMoviesAsync), HttpContext).EnrichLog();
+
+            System.Collections.Generic.List<Middleware.Validation.ValidationError> list = movieQueryParameters.Validate();
```

Comment:
I personally like declared types, but I know the IDE often suggests I shorten these to `var` when the method returns the exact type.  I'm good with either, but wanted to note it.

Reply:
when the style cop rules run in VS, it converts `var` to the correct type as part of linting